userguide/pom.xml

```diff
@@ -205,11 +205,6 @@
                     <artifactId>glassfish-doc-maven-plugin</artifactId>
                     <version>1.3</version>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-clean-plugin</artifactId>
-                    <version>3.1.0</version>
```

Comment:
this should be kept here, just remove the version number from the `build/plugins/plugin` section (around line 61 in the pom)

Reply:
I think it should be removed because there is maven clean plugin twice in `build/pluginManagement` section in the current pom. It is a mistake from first UG PR.

Reply:
there is different meaning between definition in `plugin management` and ` build plugins`. Former is like abstract definition or import if you want while the latter is concrete instance/execution

Reply:
oops, I see it's twice there. Then it is ok.